README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: method calling itself.
+- Recursive Case: a case that would cause the method to run itself
+- Base Case: the case that would cause the method to stop
+- Activation Chain/Stack: how the recursive method goes to the base case
```

Comment:
Hmm... this definition seems weird to me. Perhaps, "it traces each method call as it gets to the base case"